smart-contracts/contracts/mixins/MixinPurchase.sol

```diff
@@ -25,11 +25,13 @@ contract MixinPurchase is
 
   /**
   * @dev Purchase function
+  * @param _value the price to pay for this purchase >=keyPrice (or 0 to ignore this check)
```

Comment:
Shouldn't this be `>keyPrice`? 
if _value is `=keyPrice`, this is the same as setting `_value` to 0 isn't it?

Reply:
Setting value to keyPrice exactly doubles as a security feature.  That way if the lock owner increases the price while my transaction is pending I can't be charged more than I expected (only applicable to ERC-20 when more than keyPrice is approved for spending).

Reply:
In fact I'll add this to the comments

Reply:
Ok, makes sense. I just don't get the comment:
> (or 0 to ignore this check)
If `_value` is set to 0 the purchase will fail won't it?

Reply:
if _value is 0, you'll pay the current keyPrice

Reply:
got it, thanks

Reply:
hmm actually I should change this.  0 should mean free...

Reply:
Just read that in the PR description!

Reply:
that's going to require a lot of test updates! =p

Reply:
Hmm, if `_value` is set to `0`, you'll pay the current `keyPrice`, which would have to be `0` to indicate a free key right?

Reply:
I would think `0` only means free if it's referring to the current `keyPrice`

Reply:
i'm going to remove _value == 0 means use the current keyPrice.  always specify the value.